vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -758,6 +758,8 @@ def get_moe_wna16_block_config(config: dict[str,
 
 def should_moe_wna16_use_cuda(num_valid_tokens: int, group_size: int,
                               num_experts: int, bit: int):
+    if current_platform.is_rocm():
+        return False
     return bit == 4 and group_size in [32, 64, 128] and \
```

Comment:
Could you just add current_platform.is_cuda() to the existing conditional instead?